src/main/App.tsx

```diff
@@ -147,6 +141,15 @@ export default function App() {
   
   // "initialize" the functional component. useEffect runs before the first render
   useEffect(function () {
+    try {
+    rpcConnection = new MoneroRpcConnection({
+      uri: NODE_ADDRESS,
+      username: "username",
```

Comment:
Configuration constants should be placed clearly at the top for easy customization.